src/applications/vaos/newAppointmentFlow.js

```diff
@@ -2,13 +2,20 @@ import { getFormData } from './utils/selectors';
 
 const AUDIOLOGY = '203';
 
+function isCCAudiology(state) {
+  return (
+    getFormData(state).facilityType === 'communityCare' &&
+    getFormData(state).typeOfCareId === AUDIOLOGY
+  );
+}
+
 export default {
   home: {
     url: '/',
   },
   typeOfCare: {
     url: '/new-appointment',
-    next: 'contactInfo',
+    next: 'typeOfFacility',
```

Comment:
Assuming the user is CC eligible for now

Reply:
I think I'll write a ticket for mocking that check out. I think we know enough to do that.